package.json

```diff
@@ -3,16 +3,18 @@
   "license": "Apache-2.0",
   "packageManager": "pnpm@9.6.0+sha512.38dc6fba8dba35b39340b9700112c2fe1e12f10b17134715a4aa98ccf7bb035e76fd981cf0bb384dfa98f8d6af5481c2bef2f4266a24bfa20c34eb7147ce0b5e",
   "scripts": {
-    "format:check:all": "pnpm format:check \"+(front|shared-components|statviz)/**/*.{js,jsx,json,scss,md,ts,tsx}\" ",
+    "format:check:all": "pnpm format:check \"+(front|shared-components|statviz)/**/*.{js,json,ts,tsx}\" ",
     "format:check": "prettier --check --ignore-path .eslintignore",
-    "format:write:all": "pnpm format:write \"+(front|shared-components|statviz)/**/*.{js,jsx,json,scss,md,ts,tsx}\"",
```

Comment:
why did you remove jsx here?

Reply:
We don't use JSX ever, and if a package uses it, we don't format it ourselves.